src/domains/messages-common/04_storage.tf

```diff
@@ -80,3 +80,28 @@ resource "azurerm_key_vault_secret" "push_notifications_storage_connection_strin
 
   key_vault_id = module.key_vault.id
 }
+
+#####################################################
+module "push_notifications_storage_itn" {
+  source = "github.com/pagopa/dx//infra/modules/azure_storage_account?ref=main"
+
+  environment         = local.itn_environment
+  tier                = "l"
+  resource_group_name = "${local.prefix}-${local.env_short}-${local.location}-messages-notifications-rg"
+
+  force_public_network_access_enabled  = true
```

Comment:
I know it's currently enabled, but I would disable it in the new one.
@lucacavallaro @Garma00 sounds good?